packages/db/src/models/collection.ts

```diff
@@ -11,6 +11,14 @@ class CollectionPgModel extends BasePgModel<PostgresCollection, PostgresCollecti
       tableName: tableNames.collections,
     });
   }
+  async create(
+    knexOrTransaction: Knex | Knex.Transaction,
+    item: PostgresCollection
+  ) {
+    return await knexOrTransaction(this.tableName)
```

Comment:
There's nothing wrong with this, but since we're going to need to do the same thing for every data type, I think we should update the base model `create` to have an optional argument for `returningFields`, which could be a string or array of strings. Then from this method (and for all the other data type overrides we're going to do) we could do:

```
  return super.get(knexOrTransaction, item, '*')
```

Reply:
https://github.com/nasa/cumulus/pull/2432/commits/9ef092e76bb4d8842b815f4af0173dd7c0a61a2c
🤔